lottery/templates/lottery/signup_confirmation.html

```diff
@@ -0,0 +1,11 @@
+{% extends 'base.html' %}
+
+{% block content %}
+    <h1>Thanks!</h1>
+
+    <p>You're now in the system with email: <strong>{{ participant.email }}</strong>.</p>
```

Comment:
+ something like: `We will start inviting the first batch on X, and successive batches in the days or weeks after. *Note that if you are invited, you have 72 hours to actually confirm and pay for your ticket, or it will be assigned to someone else.`